presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -46,7 +45,7 @@
 public class ClientOptions
 {
     private static final Splitter NAME_VALUE_SPLITTER = Splitter.on('=').limit(2);
-    private static final CharsetEncoder ASCII_ENCODER = US_ASCII.newEncoder();
+    private static final CharMatcher PRINTABLE_ASCII = CharMatcher.inRange((char) 33, (char) 126);
```

Comment:
Change these to hexidecimal. It makes sense to represent ASCII ordinal as hexidecimal given how it's laid out.